MonoSynth.h

```diff
@@ -0,0 +1,22 @@
+#include "LeakedObjectDetector.h"
+#include "Oscillator.h"
+#include "Filter.h"
+#include "Amplifier.h"
+#pragma once
```

Comment:
`#pragma once` should be the first line in this file.

This line tells the compiler to only include/process the file once, no matter how many other files include it.
- [ ] revise